kansha/board/comp.py

```diff
@@ -110,7 +111,7 @@ def __init__(self, id_, app_title, app_banner, theme, search_engine,
         def many_user_render(h, number):
```

Comment:
1. Don't define functions if functions
2. Don't create functions that take `h` and generate a DOM tree: that why the views of components are made for